src/cms/views/analytics/analytics_overview.py

```diff
@@ -2,7 +2,10 @@
 from django.utils.decorators import method_decorator
 from django.views.generic import TemplateView
 
+from cms.utils.filter_links import filter_links
+
```

Comment:
```suggestion
```
`filter_links` is already imported in line 8 below. I wonder why pylint isn't complaining about this :thinking:

Reply:
This was not my intention, but both have different paths
Seems like a way to trick pylint :joy: